src/cordova/android/README.md

```diff
@@ -68,3 +68,9 @@ Using Android Studio
 - Open existing project → `<root_project_dir>/platforms/android`
 - Click on "Android Monitor" at the bottom
 - Make sure `org.outline.android.client` is selected from the middle menu (it should not say "no debuggable processes")
+
+## Recommended development flow
+
+Most Android-specific Outline code is in the `src/cordova/plugin/android/OutlineAndroidLib` directory, which is a valid Android Studio project. You can modify this code by opening this directory in Android Studio, which offers code completion, automated refactoring, version control integration, etc.
```

Comment:
From our team discussion, let's put the library at `src/cordova/android/`, since it's supposed to be independent of the plugin for the most part.

Reply:
I tried this, and it turns out it's not allowed!

> Failed to install 'cordova-plugin-outline': CordovaError: Uh oh!
File "/home/bemasc/outline-client/src/cordova/android/OutlineAndroidLib/outline" is located outside the plugin directory "/home/bemasc/outline-client/plugins/cordova-plugin-outline"